app/controllers/pages_controller.rb

```diff
@@ -1,13 +1,13 @@
 class PagesController < ApplicationController
 
   def index
-    @user =  current_user
+    @user = current_user
     if @user
       @projects = Project.where("owner = ?", @user.id)
       @announcements = Announcement.all
       @courses = @user.courses
       @assignments = @user.assignments
-      render 'dashboard' 
+      render 'dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.